### ISSUE TITLE
hemelb::redblood::orientFacets gives inconsistent results when loading deformed mesh

### ISSUE THREAD

**Issue by mobernabeu** [hemelb-codes/hemelb#735]
As part of #733, @Romain-Enjalbert found that hemelb::redblood::orientFacets will reorder 719 out 720 facets for a mesh that we know has consistent facet orientation throughout.

**Comment by rupertnash** [comment]
Hi @mobernabeu - you have added a failing test to the build. Can we remove this test as deprecated? Can we remove some code?

**Comment by mobernabeu** [comment]
Yes, we should remove the failing test and the support for .msh files altogether from the code as our implementation of `orientFacets` for them is subpar.
